server/lobbyconnection.py

```
@@ -796,6 +796,9 @@ async def command_game_join(self, message):
         assert isinstance(self.player, Player)
 
         await self.abort_connection_if_banned()
+        
+        if self.player.id in self.host.foes:
+            return;
```

No semicolon in python 😎

---

Also it might be a good idea to raise a ClientError here instead of just returning, since normally the client will expect a response when sending this command.

---

Ok awesome ill fix both of those things tomorrow.